install.sh

```diff
@@ -51,9 +59,12 @@ PREM_HOSTED_ON=docker
 PREM_AUTO_UPDATE=$PREM_AUTO_UPDATE" >$PREM_CONF_FOUND
 
     # pull latest docker compose file from main branches
```

Comment:
Add an echo to alert the user 
```suggestion
    # pull latest docker compose file from main branches
    echo "Please wait, we are downloading the latest docker compose files from $PREM_BOX_USER/$PREM_BOX_REPO/$PREM_BOX_BRANCH"

```

Reply:
Done